docs/extend/extend-apiml/authentication-for-apiml-services.md

```diff
@@ -129,40 +136,48 @@ Authentication via the login endpoint is performed in the following way:
 
 ![Zowe client certificate authentication diagram](../../images/api-mediation/zowe-client-cert-auth.png)
 
-### Procedure to authenticate with client certificates
-
-Follow these steps to authenticate with client certificates:
+#### Prerequisites
 
 1. Specify the Zowe runtime user and set your protection by password. The user is created with the `NOPASSWORD` parameter by the Zowe installer. It is necessary to change this password. For RACF, issue the following TSO command:  
 `ALTUSER <ZOWE_RUNTIME_USER (ZWESVUSR by default)> PASSWORD(<NEWPASSWORD>)`  
 For other security systems, refer to the documentation for an equivalent command.
 
+<!-- Add sample command and output -->
+
 2. Verify that the Zowe runtime user is allowed to log in to z/OSMF. (Check that the user is member of the default `IZUUSER` group.)
 
+<!-- Add how to provide this verification.. Sample commands? -->
```

Comment:
I guess this is a simple list user command

Reply:
I'm not sure I understand. Is this is the same as validating the client certificate functionality?

Reply:
No I think this is talking about the previous statement, to verify that the server user (usually zwesvusr) has access to z/OSMF by being a member of the default `IZUUSER` group

Reply:
Should I state it like this? "For RACF, issue the following TSO command to verify that zwesvusr has access to z/OSMF."?
Can you please clarify?